openstack/messaging/v2/queues/doc.go

```diff
@@ -0,0 +1,24 @@
+/*
+Package queues provides information and interaction with the queues through
+the OpenStack Messaging (Zaqar) service.
+
+Lists all queues and creates, shows information for updates, deletes, and actions on a queue.
+
+Example to Create a Queue
+
+	createOpts := queues.CreateOpts{
+		QueueName:					"My_Queue",
+		MaxMessagesPostSize: 		262143,
+		DefaultMessageTTL: 			3700,
+		DefaultMessageDelay: 		25,
+		DeadLetterQueueMessageTTL: 	3500,
+		MaxClaimCount: 				10,
+		Extra: 						map[string]interface{}{"description": "Test queue."},
+	}
```

Comment:
It's a common issue with Go and editors. `gofmt` won't format blocks of comments or strings (such as JSON fixtures). So if your editor expands/shrinks tabs/spaces, it can really alter how things look :)